google-http-client/src/main/java/com/google/api/client/util/Base64.java

```diff
@@ -26,6 +26,13 @@
  */
 @Deprecated
 public class Base64 {
+  // Guava's Base64 (https://datatracker.ietf.org/doc/html/rfc4648#section-4) decoders. When
+  // decoding, they discard the new line character so that the behavior matches what we had with
+  // Apache Commons Codec's decodeBase64. When encoding, they would insert a new line character
+  // every 64 (the 2nd argument of withSeparator method) characters. They are not used for encoding.
```

Comment:
The last word should be "decoding". :-)

Reply:
They (the two variables) are not used for encoding.

I'll rewrite this to make it clear.

Reply:
Done